packages/test-helpers/src/bytes.js

```diff
@@ -0,0 +1,13 @@
+const EMPTY_BYTES = '0x'
+const ZERO_BYTES32 =
+  '0x0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
What about `'0x' + '0'.repeat(64)`?

Reply:
Works as well! We can do so for addresses as well :).

Can you push a PR?

Reply:
Sure, I’ll do!